### DISCUSSION TITLE
Deployment on seperated domains

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [facebook/docusaurus#8482] [Q&A] [answered]
I would like to deploy my page on multiple subdomains.
What i would like is that i can have docs.mydomain.de and blog.mydomain.de.

But if possible i would also like if it would be docs.mydomain.de/xyz and not docs.mydomain.de/docs/xyz.

Is there a way to let docusaurus build a standalone blog AND standalone docs?

**Comment by slorber** [comment, marked as answer]
> What i would like is that i can have docs.mydomain.de and blog.mydomain.de.

You need 2 docusaurus sites then. It's not possible to use the same Docusaurus site across multiple domains (it is not possible in general due to building a React single-page-app)

> But if possible i would also like if it would be docs.mydomain.de/xyz and not docs.mydomain.de/docs/xyz.

You need the "docs-only" and "blog-only" mode for the 2 sites then, to remove the `routeBasePath` prefix and use `/` as the prefix instead of `/docs` or `/blog`

> Is there a way to let docusaurus build a standalone blog AND standalone docs?

Yes but you need 2 sites. You can share code between the 2 sites so that they have the exact same CSS/layout/navbar/footer if you want, so that it remains easy to maintain.
